hail-ci-build.sh

```diff
@@ -1,7 +1,38 @@
-set -x
-source activate hail
-GRADLE_OPTS=-Xmx2048m ./gradlew testAll makeDocs archiveZip --gradle-user-home /gradle-cache
+#!/bin/bash
+set -ex
+
+CLUSTER_NAME=ci-test-${SOURCE_SHA:0:12}-${SOURCE_SHA:0:12}
```

Comment:
Is the second one supposed to be `TARGET_SHA`?

Reply:
I still feel like your should generate a (globally) unique (random) cluster name.